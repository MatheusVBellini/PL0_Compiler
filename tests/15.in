PROCEDURE isOdd;
VAR num;
BEGIN
     IF ODD num THEN
     CALL printOdd;
END;

{ cometari@o }

BEGIN
     result := 0;
     i := 1;
     WHILE i <= 2 D@
          BEGIN
               IF i <> max THEN
                    result := result + 1;
               i := i + 1;
          END;
     IF result >= 50 THEN
          BEGIN
               flag := 1;
               CALL isOdd;
          END;
          flag := 0;
END. 