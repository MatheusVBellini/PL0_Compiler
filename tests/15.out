13:19 error: Missing `DO` symbol before this token
13:20 error: Invalid symbol

Compilation failed, 2 errors
